benchmarks/operator_benchmark/pt/quantization_test.py

```diff
@@ -233,6 +233,8 @@ def init(self, N, C, H, W, nbits, device, op_type):
         self.scale = torch.tensor([1.] * C).to(device)
```

Comment:
is it equivalent to `torch.ones(C, device = device)`?

Reply:
I think it would be more similar to `torch.ones(C, device=device, dtype=torch.float32)

Reply:
And similar case for torch.zeros as well around this line...

Reply:
For zero point, it will need to be `torch.zeros(C, device=device, dtype=torch.int64)` instead. Fake quantization kernel expects zero point to be of type Long.

Reply:
and more colloquial as well